### ISSUE TITLE
Stand alone html does not render code properly

### ISSUE THREAD

**Issue by neilhwatson** [arnehilmann/markdeck#13]
The slide deck below does not render the code properly in the standalone html. The indentation is lost.

   ---
   title: bug report
   pdf: slides.pdf
   standalone: slides.html

   ---

   # Code

   This code does not render prpoperly in standalone slides.html

   ```bash
   # Is indented
   if [[ 0 -eq 0 ]]
   then
      echo "do something"
   else
      echo "do something else"
   if
   ```

   ```bash
   # Looks like this in standalone html
   if [[ 0 -eq 0 ]]
   then
   echo "do something"
   else
   echo "do something else"
   if
   ```

**Comment by arnehilmann** [comment]
That is strange indeed. I try to investigate that issue in the next few days...

**Comment by arnehilmann** [comment]
fixed in arne/markdeck-standalone:0.4
*sigh* solution found here: https://stackoverflow.com/questions/29689172/bash-read-ignores-leading-spaces#29689199
